src/lrud-input/code-to-binding-map.ts

```diff
@@ -0,0 +1,12 @@
+// This maps a key code, returned from an event, to a handler name.
+// TODO: add configuration of custom key map
```

Comment:
That TODO is what I mentioned, allowing the whole keymap to be configured. As discussed this might not be necessary, so I think we can delete the TODO here